README.md

```diff
@@ -104,35 +94,15 @@ In symbols, like icons, that contain solid color backgrounds, you’ll find the
 <br />
 <br />
 
-## Recommended Sketch Plugins
-
-**Paddy**
 
-Paddy allows you to add automated padding and spacing for Sketch layers. Paddy is used throughout this Sketch document. It is not required, but is extremely helpful when adding text overrides in a symbol and having the symbol adjust to your customization with perfect padding and spacing without having to detach the symbol from this Sketch Library.
-https://github.com/DWilliames/paddy-sketch-plugin
+## Contributing, Feature Requests and Bug Reporting
 
-**Runner**
+The SLDS team welcomes your help in maintaining the SLDS Component Library. Please add any bugs or feature requests under the [Issues](https://github.com/salesforce-ux/design-system-ui-kit/issues) tab of this repository.
 
-Sketch Runner helps you to get around Sketch quicker by giving you an intuitive interface to run commands directly from your keyboard. Runner allows you to skip Sketch navigation, buttons or menus and quickly type the action you want to take like, “insert desktop tab” to add a tab component symbol to your artboard.
-[https://sketchrunner.com](https://sketchrunner.com/)
+**Public contributions are currently closed.** Salesforce designers contribute to this kit while designing Salesforce releases through an application called Abstract. These releases are completed four months before being released to the public, therefore public contributions to this kit are essentially contributing to a four month old stale release. This become unmanagable to merge Sketch binary files and isn't fair to contributors who work hard on a contribution only to have it closed without a merge.
```

Comment:
You may want this callout at the top as well

🚨 **External contributions are currently closed.** 🚨 

> Salesforce designers contribute to this kit while designing Salesforce releases through an application called Abstract. These releases are completed four months before being released to the public, therefore public contributions to this kit are essentially contributing to a four month old stale release. This become unmanagable to merge Sketch binary files and isn't fair to contributors who work hard on a contribution only to have it closed without a merge.

Reply:
Fixed